CONTRIBUTING.md

```diff
@@ -2,20 +2,20 @@
 
 ## Code of Conduct
 
-We are committed to providing a friendly, safe and welcoming environment for all. Make sure to take a look at the [Code of Conduct](CODE_OF_CONDUCT.md)!
+We are committed to providing a friendly, safe and welcoming environment for all. See our [Code of Conduct](CODE_OF_CONDUCT.md) for details.
 
 ## How to contribute
 
 All contributions are appreciated! Typo fixes, bug fixes, feature requests,
-bug reports are all helpful for the project.
+bug reports...
 
-If you are looking for a good place to start, consider reaching out on the `#contributing` channel on [Roc Zulip][roc-zulip].
-Before making your first pull request, definitely talk to an existing contributor on [Roc Zulip][roc-zulip] first about what you plan to do! This can not only avoid duplicated effort, it can also avoid making a whole PR only to discover it won't be accepted because the change doesn't fit with the goals of the language's design or implementation.
+For a good place to start, consider reaching out on the `#contributing` channel on [our zulip group chat][roc-zulip].
+Before making your first pull request, talk to an existing contributor on [zulip][roc-zulip] about what you plan to do! This can avoid duplicated effort or a rejection because the change doesn't fit with the goals of the language.
```

Comment:
I've made this document more compact so that people are more likely to read it.

Reply:
It's definitely more succinet and clearer